addons/web/static/src/js/views/search_panel.js

```diff
@@ -272,13 +272,20 @@ const SearchPanel = Widget.extend({
     _createCategoryTree(categoryId, values) {
         const category = this.categories[categoryId];
         const parentField = category.parentField;
+        const unfoldedIds = [];
+        if (category.values) {
+            const unfolded = Object.values(category.values)
+                .filter(c => c.folded === false)
+                .map(c => c.id);
+            unfoldedIds.push(...unfolded);
+        }
 
         category.values = {};
         values.forEach(value => {
             category.values[value.id] = Object.assign({}, value, {
                 childrenIds: [],
-                folded: true,
-                parentId: value[parentField] && value[parentField][0] || false,
+                folded: !unfoldedIds.includes(value.id),
+                parentId: value[parentField] || false,
```

Comment:
yet another change done by this commit right? All of them must be describer in the commit message

Reply:
ok I understand now, it's fine